main/gps.go

```diff
@@ -10,23 +10,34 @@
 package main
```

Comment:
All NPEA parser are now separated into its own file. (gps_nmeaparsers.go) This is to ensure we parse a NMEA sentence without updating stratux. THis is needed because we need to inspect each GPS nmea sentence and look for specific information needed to decide the reception status of the GPS.